app/Repositories/LmsSetting/LmsSettingRepository.php

```diff
@@ -32,9 +32,13 @@ public function getAll($data)
         $perPage = isset($data['size']) ? $data['size'] : config('constants.default-pagination-per-page');
         $query = $this->model;
 
-        // if specific index projects requested
         if (isset($data['query']) && $data['query'] !== '') {
-            $query = $query->where('lms_url', 'iLIKE', '%'.$data['query'].'%');
+            $query = $query->whereHas('user', function ($qry) use ($data) {
+                $qry->where('first_name', 'iLIKE', '%' .$data['query']. '%');
+                $qry->orWhere('last_name', 'iLIKE', '%' .$data['query']. '%');
+                $qry->orWhere('email', 'iLIKE', '%' .$data['query']. '%');
+            });
+            $query = $query->orwhere('lms_url', 'iLIKE', '%'.$data['query'].'%');
```

Comment:
Please Add Blank space before and after "." concatenation operator

Reply:
Done